crowbar_framework/lib/remote_node.rb

```diff
@@ -64,11 +64,11 @@ def self.chef_clients_running?(host_or_ip)
     end
   end
 
-  def self.reboot_done?(host_or_ip, reboot_requesttime)
+  def self.reboot_done?(host_or_ip, reboot_requesttime, options)
```

Comment:
if not needed, we will have to adapt the [spec](https://github.com/crowbar/barclamp-crowbar/blob/master/crowbar_framework/spec/lib/remote_node_spec.rb) i guess